docs/grafanacloud.md

```diff
@@ -0,0 +1,185 @@
+
+This walkthrough is intended for use by people who would like to deploy Grafana and the OCI Data Source for Grafana in **Grafana Cloud**.
+
+Make sure you have access to the [Monitoring Service](https://docs.cloud.oracle.com/iaas/Content/Monitoring/Concepts/monitoringoverview.htm) and that [metrics have been enabled](https://docs.cloud.oracle.com/iaas/Content/Compute/Tasks/enablingmonitoring.htm) for the resources you are trying to monitor.
+
+## Getting OCI Configuration values
+
+To configure OCI Metrics Grafana Data Source, you'll need to get the necessary provider and resource settings. Please note that with **Grafana Cloud** only version 4.x.x and above are supported. If you are configuring the plugin to work in Multitenancy Mode, you will need to repeat the following steps for each of the Tenancies you want to configure with the plugin (up to 5 additional Tenancies are supported).
+
+### Getting the Region
+
+To get the region for your OCI cloud, follow these steps:
+
+1. Log in to the OCI console.
+2. From the OCI menu, select the **Region** dropdown in the top right corner of the page.
+3. The region is listed next to **Home**.
+
+For details and reference, see: [Regions and Availability Domains](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm#top)
+Make note of the region as you'll need it later to configure your OCI Metrics Grafana Data Source.
+
+### Getting the Tenancy OCID
+
+To get the tenancy OCID, follow these steps:
+
+1. Log in to the OCI console.
+2. From the OCI menu, click on your profile icon on the top right:
+
+![OCI Administration](images/oci_administration.png)
+
+3. Click on Tenancy
+4. The tenancy OCID is listed in the **Tenancy Information** section.
+
+![OCI Tenancy](images/oci_tenancy.png)
+
+For details and reference, see: [Where to Get the Tenancy's OCID and User's OCID](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#five)
+Make note of the tenancy OCID as you'll need it later to configure your OCI Metrics Grafana Data Source.
+
+### Getting the User OCID
+
+To get the user OCID, follow these steps:
+
+1. Log in to the OCI console.
+2. From the OCI menu, select **Identity** > **Users**.
+3. Click on the user you want to use with OCI Metrics Grafana Data Source.
+4. The user OCID is listed in the **User Details** section.
+
+![OCI User](images/oci_user.png)
+
+For details and reference, see: [Where to Get the Tenancy's OCID and User's OCID](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#five).
+Make note of the user OCID as you'll need it later to configure your OCI Metrics Grafana Data Source.
+
+### Getting the API Key Fingerprint
```

Comment:
It would be more logical to include these steps as part of the next section rather than a completely independent section since the steps are basically identical and relate to the API keys.